app/controllers/api/memberships_controller.rb

```diff
@@ -43,8 +55,8 @@ def remove_admin
   private
 
   def visible_records
-    load_and_authorize :group
-    Queries::VisibleMemberships.new(user: current_user, group: @group)
+    resource_class.includes(:user, :group, :inviter)
+                  .where("group_id IN (#{current_user.group_ids.join(',')}) OR groups.is_visible_to_public = 't'")
```

Comment:
A group is visible to a user if a) they're in it, or b) it is visible to the public. Seems legit?